### DISCUSSION TITLE
extension dissapeared

### DISCUSSION THREAD

**Discussion by frozenparagon** [austencm/youtube-auto-like#121] [General]
seems like something on the computer removed the extension without confirmation at some point. it's since been reinstalled. I've never gone digging in chrome logs, but if you aim me a bit I'd be happy to look for what may have removed the extension and perhaps you can look into it (author)

**Comment by austencm** [comment]
Did the folder it was in get deleted or moved? Or is Chrome just forgetting about it? If the latter, this could be the same issue as #116. 

Also, what's your OS?

**Comment by frozenparagon** [comment]
OH. oh. windows 10 and storage sense is just doing it's job. It purges old content in my downloads folder. I see now. I forget that unpacked loads don't save in a central location do they? they just live where they're at. easy fix, I'll move folder to a safe location. thx for reply, this inevitably happening a second time would've annoyed me

this extension is fantastic btw. i've been telling folks about it

**Comment by austencm** [comment]
Yep that should fix it! :)

**Comment by frozenparagon** [comment]
any interest in adjusting the project folder layout so the github local copy can be loaded as the extension?
it would specifically allow for live updaing

**Reply by austencm** [reply]
tracking webpack-generated files feels wrong, but I see how it could be worth it in this case. I’ll add it to the list
